contrib/omhttp/omhttp.c

```diff
@@ -3,6 +3,10 @@
  *
  * NOTE: read comments in module-template.h for more specifics!
  *
+ * Supports profile-based configuration for common HTTP endpoints:
+ * - profile="loki" for Grafana Loki
+ * - profile="hec:splunk" for Splunk HTTP Event Collector
```

Comment:
pls mention that hec is only proof-of-concept, mention we wait for feedback from https://github.com/rsyslog/rsyslog/pull/5756 (be sure to include url)